src/Pages/About/Safarri/Safarri.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Saf1 from '../../../asserts/images/saf1.png';
```

Comment:
When importing files you want to give them the right names. Saf1, Saf2, Saf3 are all not valid here